server/controllers/article/get-all.php

```diff
@@ -33,11 +33,11 @@ public function validations() {
     public function handler() {
         $topics = Topic::getAll();
         $topicsArray = [];
-        
+
         foreach($topics as $topic) {
-            $topicsArray[] = $topic->toArray();
+            Controller::isStaffLogged() ? $topicsArray[] = $topic->toArray() : ($topic->private*1 ? null : $topicsArray[] = $topic->toArray())  ;
```

Comment:
Could you use an `if` so it's more clear?
Also, `*1` casting is not necessary here. 
```suggestion
            if (Controller::isStaffLogged()) {
                $topicsArray[] = $topic->toArray();
            } else if (!$topic->private) {
                $topicsArray[] = $topic->toArray();
            }
```